assets/js/modules/tagmanager/components/settings/SettingsView.js

```diff
@@ -255,7 +264,7 @@ export default function SettingsView() {
 					</p>
 
 					{ hasExistingTag && (
-						<P>
+						<P className="zbi">
```

Comment:
@abdelmalekkkkk, what is the purpose of this class? Let's use the existing class to appy `margin-bottom: 0`:
```suggestion
						<P className="googlesitekit-margin-bottom-0">
```